mathcheck: error when compiled with icc

See http://community.markus-raab.org:8080/job/elektra-icc/lastFailedBuild/console, mathcheck seems to have issues when compiled with icc?

```
 22/106 Test  #22: testmod_mathcheck ..................***Failed    0.50 sec
elektraFtoA: val: 1,5, ret: 1.5
elektraEFtoF: string: 1.5, ret: 1,5
MATHCHECK	   TESTS
==================

/home/jenkins/workspace/workspace/elektra-icc/src/plugins/mathcheck/testmod_mathcheck.c:73: error in main: error
/home/jenkins/workspace/workspace/elektra-icc/src/plugins/mathcheck/testmod_mathcheck.c:77: error in main: error
/home/jenkins/workspace/workspace/elektra-icc/src/plugins/mathcheck/testmod_mathcheck.c:81: error in main: error
/home/jenkins/workspace/workspace/elektra-icc/src/plugins/mathcheck/testmod_mathcheck.c:85: error in main: error
/home/jenkins/workspace/workspace/elektra-icc/src/plugins/mathcheck/testmod_mathcheck.c:89: error in main: error

testmod_mathcheck RESULTS: 60 test(s) done. 5 error(s).
```

--- a/src/plugins/mathcheck/testmod_mathcheck.c
+++ b/src/plugins/mathcheck/testmod_mathcheck.c
@@ -15,16 +15,16 @@
 
 #include <tests_plugin.h>
 
-#define test(ks, retval)																												   \
-	{																																   \
-		Key * parentKey = keyNew ("user/tests/mathcheck", KEY_VALUE, "", KEY_END);												   \
-		KeySet * conf = ksNew (0, KS_END);																						   \
-		PLUGIN_OPEN ("mathcheck");																								   \
-		ksRewind (ks);																											   \
-		succeed_if (plugin->kdbSet (plugin, ks, parentKey) == retval, "error");													   \
-		keyDel (parentKey);																										   \
-		PLUGIN_CLOSE ();																										   \
-	}
+static inline void test (KeySet * ks, int retval)
+{
+	Key * parentKey = keyNew ("user/tests/mathcheck", KEY_VALUE, "", KEY_END);
+	KeySet * conf = ksNew (0, KS_END);
+	PLUGIN_OPEN ("mathcheck");
+	ksRewind (ks);
+	succeed_if (plugin->kdbSet (plugin, ks, parentKey) == retval, "error");
+	keyDel (parentKey);
+	PLUGIN_CLOSE ();
+}
 
 static KeySet * create_ks (const char * res, const char * meta)
 {
@@ -78,15 +78,15 @@ int main (int argc, char ** argv)
 	ksDel (ks); 
 
 	ks = create_ks ("3", "== + bla/nonExisting + bla/nonExistingToo bla/val3");
-	test (ks, 1)
+	test (ks, 1);
 	ksDel (ks);   
   
 	ks = create_ks ("3", "== / bla/nonExisting / bla/nonExistingToo bla/val3");
-	test (ks, 1)
+	test (ks, 1);
 	ksDel (ks);   
   
 	ks = create_ks ("3", "== + bla/nonExisting / bla/nonExistingToo bla/val3");
-	test (ks, 1)
+	test (ks, 1);
 	ksDel (ks);   
 
 	printf ("\ntestmod_mathcheck RESULTS: %d test(s) done. %d error(s).\n", nbTest, nbError);

